pyproject.toml

```diff
@@ -105,112 +109,104 @@ style = "pep440"
 vcs = "git"
 
 [tool.poetry-dynamic-versioning.substitution]
-files = ["pycytominer/__about__.py"]
+files = [ "pycytominer/__about__.py" ]
 
-[tool.commitizen]
-# This version is used for changelog tracking and is updated using `cz bump`
-version = "1.2.0"
-name = "cz_conventional_commits"
-tag_format = "v$version"
-version_scheme = "pep440"
-version_provider = "commitizen"
-update_changelog_on_bump = true
-template = "dev_tools/commitizen/CHANGELOG.md.j2"
-version_files = [
-    "CITATION.cff"
-]
+[tool.setuptools_scm]
+# Avoids warnings aboud missing section
```

Comment:
Typo?

Reply:
Ah good catch - I'll get this corrected!